.github/workflows/main.yaml

```diff
@@ -51,10 +63,11 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@171dd555b9ab6b18fa02519fdfacbb8bf671e1b4  # locking to a commit version for now
         with:
-          branch: automation/add-new-dotnet-release
-          title: New DotNetCore Sdks
+          branch: automation/add-new-dotnet-release-${{ env.releaseDate }}
```

Comment:
nit: how does appending the `releaseDate` input to the branch affect the number of PRs that will be opened? Will this logic continue to push to the same branch that a PR is opened for, or will one be opened every 12 hours while an SDK change is needed to be made?

Reply:
If .NET releases a new 7.0 version, and our noon cron job picks up the change and creates a PR based on it, but that PR isn't merged by the end of the day, would the nightly cron job pick up the same change from .NET (since it's not yet in our repository) and try to create another branch based on the same changes that are already in the PR?

Reply:
In that case it would use the same branch and PR.

The problem I do see is if it get's merged to `main` but the `Oryx-PublishSdkToProd` doesn't get triggered before the nightly cron. Then it would re-open the same PR on the same branch. We might have just have to keep the nightly cron.